skimage/util/_montage.py

```diff
@@ -0,0 +1,252 @@
+import numpy as np
+from .. import exposure
+from .._shared.utils import deprecated
+
+
+__all__ = ['montage', 'montage2d']
+
+
+def montage(arr_in, fill='mean', rescale_intensity=False, grid_shape=None,
+            padding_width=0, multichannel=False):
+    """Create a 2-dimensional 'montage' from a 3-/4-dimensional input array
```

Comment:
2-dimensional is overloaded here; perhaps "Create a montage / rectangular collage" instead of "2-dimensional 'montage'".  The dimensionality here gets confusing when, directly after, 3-/4-dimensional is mentioned.

This also does not correspond to the single line, single sentence docstring format.

Suggested:

"Create a montage of several grey or color images."

And then a longer second paragraph.